### PR TITLE
Enable LZO for bacula-fd

### PR DESCRIPTION
**Pull request by matlink**

### CODE REVIEW

**Comment by 2096779623** [comment]
Why split into two PR?

**Comment by matlink** [comment]
> Why split into two PR?

One for enabling LZO, one for upgrading bacula.

File: packages/bacula-fd/build.sh [hunk 1]
```diff
@@ -3,10 +3,11 @@ TERMUX_PKG_DESCRIPTION="Bacula backup software"
 TERMUX_PKG_LICENSE="AGPL-V3"
 TERMUX_PKG_MAINTAINER="Matlink <matlink@matlink.fr>"
 TERMUX_PKG_VERSION=11.0.5
-TERMUX_PKG_REVISION=3
+TERMUX_PKG_REVISION=4
 TERMUX_PKG_SRCURL=https://sourceforge.net/projects/bacula/files/bacula/${TERMUX_PKG_VERSION}/bacula-${TERMUX_PKG_VERSION}.tar.gz
 TERMUX_PKG_SHA256=ef5b3b67810442201b80dc1d47ccef77b5ed378fe1285406f3a73401b6e8111a
-TERMUX_PKG_DEPENDS="openssl"
+TERMUX_PKG_DEPENDS="openssl, liblzo"
+TERMUX_PKG_BUILD_DEPENDS="liblzo"
```
Comment (Grimler91) [inline packages/bacula-fd/build.sh:10]: No need to add liblzo to both BUILD_DEPENDS and DEPENDS, DEPENDS should be enough and correct

**Review by Grimler91 — CHANGES_REQUESTED** [review]

File: packages/bacula-fd/build.sh (hunk 1 above)
Comment (matlink) [reply packages/bacula-fd/build.sh]: Should be good now

**Comment by 2096779623** [comment]
Don't update upstream like this.
First use `git reset` to roll back to the previous commit.
Use `git remote add upstream https://github.com/termux/termux-packages && git pull upstream master --rebase && git push origin bacula-lzo -f` to make it upto date.
Or
image

**Comment by matlink** [comment]
> Don't update upstream like this.

Is it good now?

**Comment by 2096779623** [comment]
> Is it good now?

Yeah.

**Review by 2096779623 — APPROVED** [review]

**Pull request merged**